eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala

```diff
@@ -48,12 +50,14 @@ object ExplorerApi {
   implicit val formats: DefaultFormats = DefaultFormats
   implicit val serialization: Serialization = Serialization
 
+  import fr.acinq.bitcoinscala.KotlinUtils.scala2kmp
+
   sealed trait Explorer {
     // @formatter:off
     /** Explorer friendly-name. */
     def name: String
     /** Map from chainHash to explorer API URI. */
-    def baseUris: Map[ByteVector32, Uri]
+    def baseUris: Map[ByteVector32Kmp, Uri]
```

Comment:
The pb is that the type of all genesis block hashes is `fr.acinq.bitcoin.ByteVector32`,  and we use them as keys to `baseUris`

Reply:
It doesn't have to be, you can simply import `fr.acinq.bitcoinscala.Block` instead of `fr.acinq.bitcoin.Block` to get the genesis block hashes (which are then `ByteVector32` from the `bitcoinscala` package).

Just replace your import from `fr.acinq.bitcoin.Block` to `fr.acinq.bitcoinscala.Block` and you don't need to change this line (and the others mentioned in https://github.com/ACINQ/eclair/pull/2038#discussion_r803519256)